### DISCUSSION TITLE
Birdseye view automatically picking up browser dimensions?

### DISCUSSION THREAD

**Discussion by rknobbe** [blakeblackshear/frigate#9054] [Ideas]
I run one of my monitors in portrait view, the other in landscape view. Birdseye always lays out camera feeds in a fixed ratio landscape orientation. Can it be enhanced to lay out based on the browser's height and width and pack the window appropriately?

**Comment by NickM-27** [comment]
Birdseye is encoded on the frigate side not in the browser. This wouldn't be easy to do especially supporting multiple clients at once. You can configure birdeye resolution
